src/main/java/duke/exceptions/InvalidTaskNumberException.java

```diff
@@ -0,0 +1,11 @@
+package duke.exceptions;
+
+public class InvalidTaskNumberException extends DukeException {
```

Comment:
Applies to all custom exceptions you have created:
Since it is a custom exception, it becomes imperative to explain why you need this via a header comment.